epsilon-runtime-osgi/pom.xml

```diff
@@ -16,6 +16,8 @@
             <plugin>
                 <groupId>org.apache.felix</groupId>
                 <artifactId>maven-bundle-plugin</artifactId>
+                <version>5.1.6</version>
```

Comment:
Shouldn't this version come from the parent pom?

Reply:
Idea have no idea what is the version....